bazel/repository_locations.bzl

```diff
@@ -259,6 +259,14 @@ DEPENDENCY_REPOSITORIES = dict(
         use_category = ["dataplane"],
         cpe = "cpe:2.3:a:gnu:zlib:*",
     ),
+    com_github_zlib_ng_zlib_ng = dict(
+        sha256 = "9646501422faf7a0b3a1d82402a421c0d56c9f4e1926692b6ed0bd80042da209",
+        strip_prefix = "zlib-ng-c71ca170224f31a219513aa470f82aa50a3ded48",
+        # 2020-07-31 develop branch.
```

Comment:
nit: development

Reply:
The default branch on the repo is actually called `develop` - https://github.com/zlib-ng/zlib-ng/branches

Reply:
Ah -- OK.